branchLayoutImpl/src/main/java/com/virtuslab/branchlayout/impl/readwrite/BranchLayoutFileWriter.java

```diff
@@ -50,10 +51,12 @@ public void write(Path path, IBranchLayout branchLayout, boolean backupOldFile)
       @NonNegative int level) {
 
     List<String> stringList = List.empty();
-    for (var entry : entries) {
-      var sb = new StringBuilder();
-      sb.append(String.valueOf(indentSpec.getIndentCharacter()).repeat(level * indentSpec.getIndentWidth()))
-          .append(entry.getName());
+    for (val entry : entries) {
+      val sb = new StringBuilder();
+      for (int i = 0; i < level * indentSpec.getIndentWidth(); i++) {
```

Comment:
Meh

Reply:
What about extracting `level * indentSpec.getIndentWidth()` to the `{lombok.|}val`?

---

https://stackoverflow.com/questions/3881660/java-for-loop-evaluation